src/bin/download_sysext.rs

```diff
@@ -108,6 +112,34 @@ impl<'a> Package<'a> {
             return true;
         }
     }
+
+    fn verify_signature_on_disk(&mut self, from_path: &Path) -> Result<(), Box<dyn Error>> {
+        const PUBKEY_FILE: &str = "../src/testdata/public_key_test_pkcs8.pem";
```

Comment:
should take the path to the pubkey as a function argument rather than having it hard-coded

Reply:
Ok, done.